Formula/neko.rb

```diff
@@ -14,6 +14,13 @@ class Neko < Formula
       url "https://github.com/HaxeFoundation/neko/commit/a8c71ad97faaccff6c6e9e09eba2d5efd022f8dc.patch?full_index=1"
       sha256 "a5d08e5ff2f6372c780d2864b699aae714fc37d4ab987cea11764082757ddb39"
     end
+
+    patch do
```

Comment:
Is this Xcode 9 only? If so it should be scoped to `if DevelopmentTools.clang_build_version >= 900`

Reply:
I have no idea why it previously worked. But it currently doesn't build without this patch for my MacBook, which is using macOS 10.13, XCode 9.2. I tried to reproduce it in Travis using its [XCode9.2 image (it's macOS 10.12)](https://docs.travis-ci.com/user/reference/osx/), but the build passed: https://travis-ci.org/HaxeFoundation/neko/builds/315224085

Reply:
It appears fine on Jenkins: https://jenkins.brew.sh/job/Homebrew%20Testing/391/

Reply:
umm... I'm not sure what is the exact cause of https://github.com/HaxeFoundation/neko/issues/173, but at least the patch wouldn't break earlier version of Neko/macOS/XCode.